### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by ks0002544** [jellyfin/jellyfin-tizen#237]
this is not a problem but a wish
After deleting the viewed item, the page returns to the main menu. Is it possible to stay on the page from which the deletion occurred or to automatically re-read it?

**Comment by penguintamer** [comment]
I think you want the jellyfin-web repository. Jellyfin-tizen is just for the Tizen release, which is based on jellyfin-web. https://github.com/jellyfin/jellyfin-web/issues

**Comment by ks0002544** [comment]
yes I am using Jellyfin on my Samsung TV Tizen. Question was regarding Jellyfin tizen

**Comment by dmitrylyzo** [comment]
> yes I am using Jellyfin on my Samsung TV Tizen. Question was regarding Jellyfin tizen

Tizen app is just a wrapper of jellyfin-web. That's why any changes not directly related to TV should be made there.
